common/HeaderSlim.tsx

```diff
@@ -39,6 +40,13 @@ export const HeaderSlim: React.FC<Props> = ({
     if (anchor !== tab) setTab(anchor || 'browse')
   }, [router.asPath, tab])
 
+  useEffect(() => {
+    if (wallet.connected && wallet.publicKey) {
+      const userId = wallet.publicKey.toString()
+      amplitude.setUserId(userId)
+    }
+  }, [wallet.connected, wallet.publicKey])
```

Comment:
wallet.publicKey needs to be wallet.publicKey.toString() because publicKey will cause this to run every render

also maybe we should use `useWalletId`? do we want this to show up as the user if i add ?wallet={someoneElsesWallet}

Reply:
also curious why in HeaderSlim